base_bank_from_iban/tests/test_base_bank_from_iban.py

```diff
@@ -1,5 +1,5 @@
 # Copyright 2017 Tecnativa - Carlos Dauden
-# Copyright 2022 Tecnativa - Pedro M. Baeza
+# Copyright 2022,2024 Tecnativa - Pedro M. Baeza
```

Comment:
Is this needed or not?:
https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0#tasks-not-to-do-in-the-migration

Reply:
Well, I have adapted the format to not put as ending date, but "continuing" date. I want to highlight that it has been touched in these 2 years.

Reply:
So what should I put in a module that for example has been played in the years: 2016, 2017, 2020 and 2023?

Reply:
I would put 2016,2023

Reply:
And when would the usual 2016-2023 be used?

Reply:
Well, @sbidoul put that note because he says that putting that notation means that the copyright ends on that date, which is right, so I have somehow invented this variant to highlight some work between several years, only putting the first and the last you have touched it.